### ISSUE TITLE
Xdg base directory specification support

### ISSUE THREAD

**Issue by bijang** [stumpwm/mahogany#4]
Stupid question probably, but is this supported, for config dot file allocation etc?
Only thought of it when I see emacs 27 is adopting it.

**Comment by sdilts** [comment]
Yep. Unless there are loud objections, I don't even think we will support having a `.stumpwm.d` directory or other alternatives.

**Comment by sdilts** [comment]
Hi @bijang, we got basic configuration file support added, and it follows the XDG specification. There are no APIs for customization yet, but it's in there!
